Tests/SwiftSyntaxTest/BumpPtrAllocatorTests.swift

```diff
@@ -0,0 +1,43 @@
+#if DEBUG
+// 'BumpPtrAllocator' is not a public API. Test it only when the module is built
+// with @testable (i.e. `DEBUG`)
```

Comment:
I’m not sure if this is a great idea because it means that these tests won’t be run in CI. Why are you so afraid to mark `BumpPtrAllocator` as `@_spi(Testing)`

Reply:
It was just a personal preference. Updated to `@_spi(Testing)`

Reply:
We disable testable imports in CI because it would cause us to rebuild the entire package when testing it after we have already built it in a first step.